flujo/recipes/factories.py

```
@@ -411,17 +411,60 @@ async def run_default_pipeline(
-            break
+    solution = None
+    checklist: Checklist | None = None
+
+    try:
+        # Primary sources filled by step callables
+        if context is not None and hasattr(context, "scratchpad"):
+            solution = context.scratchpad.get("solution")
+            # Do not take checklist from scratchpad yet; prefer validator output below
+    except Exception:
+        # Best-effort: proceed to fallbacks below
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)